frontend/src/components/ToDoLists.js

```diff
@@ -64,6 +64,16 @@ export const ToDoLists = ({ userState }) => {
             setSelectedList(res)
         }
     }
+    // handler for deleting list
+    const handleDeleteList = async (listId) => {
+        if (!window.confirm("Are you sure?")) {
+            return;
+        }
+        await deleteList(listId);
```

Comment:
Why don't we check the response? The response should be a 204 (success) or an error, correct?